lib/index.js

```diff
@@ -40,20 +40,25 @@ if (typeof process.env.NODE_PG_FORCE_NATIVE !== 'undefined') {
   module.exports = new PG(Client)
 
   // lazy require native module...the native module may not have installed
-  module.exports.__defineGetter__('native', function () {
-    delete module.exports.native
-    var native = null
-    try {
-      native = new PG(require('./native'))
-    } catch (err) {
-      if (err.code !== 'MODULE_NOT_FOUND') {
-        throw err
+  var native
+  Object.defineProperty(module.exports, 'native', {
+    enumerable: false,
```

Comment:
Unconfigurable properties are rarely necessary, and with `enumerable: false` being the default:

```suggestion
    configurable: true,
```

Reply:
Updated